uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
+        def staffers_by_job_options(self, job):
+            """
+            Format staffers_by_job() for use with the {% options %} template decorator
+            """
+            return [(a.id, a.full_name) for a in self.staffers_by_job(job)]
+
```

Comment:
Couple of super-minor things.  Like, minor enough I wouldn't even bother changing anything unless you felt compelled to.

1) I would be inclined to put this as a method or even a property on the ``Job`` class.  When a model object is bound to a session, you can access the session with the ``.session`` property, so you could say something like

```python
class Job(MagModel):
    ...
    @property
    def available_staffers_or_whatever_you_would_call_this(self):
        return self.session.query(Attendee) ...
```

2) When you've got 4 lines in a row that need to end with a backslash I'd generally just wrap the whole thing in parenthesis, e.g.

```python
return (self.query(Attendee)
            .filter_by(staffing=True)
            ...)
```

but that's just a stylistic preference.

3) You can use lists as well as tuples with the ``*`` syntax, which would look a lot cleaner in this case.  I'd write that filter line as

```python
.filter(*[Attendee.trusted_depts.contains(str(job.location))] if job.restricted else [])
```
which to my eyes looks a lot cleaner, since single-item tuples use that ugly ``('one item',)`` syntax.  Bleh.